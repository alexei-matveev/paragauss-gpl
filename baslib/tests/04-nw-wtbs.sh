../bin/qm-find-basis --library=nwchem li wtbs ahlrichs_coulomb_fitting
