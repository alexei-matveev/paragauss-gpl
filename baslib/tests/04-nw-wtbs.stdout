&unique_atom_basisset
lmax_ob=0
lmax_ch=3
lmax_xc=0
lmax_pseudo=-1
zc=0
/
&unique_atom_basis
n_exponents=20
n_contracted_fcts=2
n_uncontracted_fcts=0
/
0.017483139 0.035101438 0.070475145 0.14150657 0.28419885 0.57116237 1.1496894 2.3217784 4.7178658 9.6913182 20.267021 43.58338 97.686858 232.152 597.02314 1699.6534 5484.1328 20515.098 90788.61 483307.72
-0.118268922 -0.47634812 -0.429047357 -0.107720377 0.0603857299 0.0927545288 0.0755493709 0.0519926103 0.0311065765 0.0162748135 0.00755701887 0.00322959721 0.00128632742 4.78032183e-4 1.62207357e-4 4.90295015e-5 1.27351664e-5 2.7716815e-6 4.76665013e-7 7.59166201e-8
4.71149854e-5 3.9082103e-4 1.30139383e-4 8.33426685e-4 0.0350450457 0.186976776 0.306473526 0.273481058 0.180686934 0.0988177384 0.0471582355 0.0204004785 0.00819172329 0.00305101345 0.00103763269 3.1362762e-4 8.15223537e-5 1.77366412e-5 3.05140361e-6 4.85854775e-7
&unique_atom_basis
n_exponents=0
n_contracted_fcts=0
n_uncontracted_fcts=0
/
&unique_atom_basis
n_exponents=9
n_contracted_fcts=7
n_uncontracted_fcts=0
/
0.069296 0.151659 0.336355 0.775515 1.90155 5.052147 14.753627 47.834436 173.289709
0.0 0.0 0.0 0.0 0.0 0.0 0.66520903 0.25220565 0.08014875
0.0 0.0 0.0 0.0 0.0 1.0 0.0 0.0 0.0
0.0 0.0 0.0 0.0 1.0 0.0 0.0 0.0 0.0
0.0 0.0 0.0 1.0 0.0 0.0 0.0 0.0 0.0
0.0 0.0 1.0 0.0 0.0 0.0 0.0 0.0 0.0
0.0 1.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0
1.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0
&unique_atom_basis
n_exponents=2
n_contracted_fcts=2
n_uncontracted_fcts=0
/
0.159554 0.377864
0.0 1.0
1.0 0.0
&unique_atom_basis
n_exponents=2
n_contracted_fcts=2
n_uncontracted_fcts=0
/
0.237378 0.560493
0.0 1.0
1.0 0.0
&unique_atom_basis
n_exponents=1
n_contracted_fcts=1
n_uncontracted_fcts=0
/
0.468426
1.0
&unique_atom_basis
n_exponents=0
n_contracted_fcts=0
n_uncontracted_fcts=0
/
&unique_atom_basis
n_exponents=0
n_contracted_fcts=0
n_uncontracted_fcts=0
/
